packages/design-system/.eslintrc

```diff
@@ -8,5 +8,6 @@
     // TODO: re-enable this rules
     "react/display-name": "off",
     "@typescript-eslint/no-explicit-any": "warn",
+    "import/export": "off",
```

Comment:
Why did you add this?

Reply:
typescript manages this for us, and there were weird errors where eslint thought there was an issue with dual exports but it wasn't an issue since we use TS namespacing 🤔